src/database/database.module.ts

```diff
@@ -0,0 +1,17 @@
+import { Module } from '@nestjs/common';
```

Comment:
From my point of view it is not necessary to create a module for the database since it is something that will only be used in app.module.
You can do something like this in app.module

```
@Module({
  imports: [
    ConfigModule.forRoot({
      isGlobal: true,
    }),
    TypeOrmModule.forRoot({
      type: 'postgres',
      host: process.env.DB_HOST,
      port: Number(process.env.DB_PORT),
      username: process.env.DB_USERNAME,
      password: process.env.DB_PASSWORD,
      database: process.env.DB_NAME,
      schema: 'public',
      entities: ['dist/**/*.entity.js'],
      migrationsTableName: 'migrations_history',
      migrations: ['dist/migration/*.js'],
      migrationsRun: !!process.env.DB_MIGRATE,
      cli: {
        migrationsDir: 'migration',
      },
    }),
  ],
  controllers: [AppController],
  providers: [AppService],
})
```